src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -773,6 +796,32 @@ export class OnlineGameWrapperComponent extends GameWrapper implements OnInit, O
         }
         return true;
     }
+    public addGlobalTime(): Promise<void> {
+        const giver: Player = Player.of(this.observerRole);
+        return this.gameService.addGlobalTime(this.currentPartId, this.currentPart, giver);
+    }
+    public addLocalTime(): Promise<void> {
+        const giver: Player = Player.of(this.observerRole);
+        return this.gameService.addLocalTime(giver, this.currentPartId);
+    }
+    public addLocalTimeTo(player: Player, addedMs: number): void {
+        if (player === Player.ZERO) {
+            const currentDuration: number = this.chronoZeroLocal.remainingMs;
```

Comment:
Change chronoZeroLocal to an array of chronoLocal[player]? (or ticket it)

Reply:
ticketté!